# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/PHttpBackgroundChannel.ipdl
# Commit: 2a9873ad6856
# Full Hash: 2a9873ad68561b991cef5795e239a571f08e5aac
# Author: Junior Hsu <juhsu@mozilla.com>
# Date: 2020-06-24 09:31:07
# Regressor Bug: 1633935
# File Overlap Count: 1
# Description:
#   Bug 1633935 - P4 Remove On[Start|Stop]Request/OnTransportAndData in PHttpChannel, r=mayhemer,nika,necko-reviewers
#   
#   We move OnStartRequest from PHttpChannel to PHttpBackgroundChannel, thus adjusting
#   message-metadata.ini
#   
# ==============================================================================

diff -r be03e0e7b645 -r 2a9873ad6856 netwerk/protocol/http/PHttpBackgroundChannel.ipdl
--- a/netwerk/protocol/http/PHttpBackgroundChannel.ipdl	Tue Jun 23 16:56:45 2020 +0000
+++ b/netwerk/protocol/http/PHttpBackgroundChannel.ipdl	Tue Jun 23 16:56:53 2020 +0000
@@ -27,9 +27,6 @@
 
   // Combines a single OnDataAvailable and its associated OnProgress &
   // OnStatus calls into one IPDL message
-  // This is duplicated on PHttpChannel, which gets used for multi-part
-  // streams to make synchronization when we get OnStartRequest multiple
-  // times easier.
   async OnTransportAndData(nsresult  channelStatus,
                            nsresult  transportStatus,
                            uint64_t  offset,
@@ -37,9 +34,7 @@
                            nsCString data,
                            bool dataFromSocketProcess);
 
-  // This is duplicated on PHttpChannel, which gets used for multi-part
-  // streams to make synchronization when we get OnStartRequest multiple
-  // times easier.
+
   async OnStopRequest(nsresult channelStatus,
                       ResourceTimingStructArgs timing,
                       TimeStamp lastActiveTabOptimization,